.travis.yml

```diff
@@ -7,5 +7,12 @@ services:
 script:
   - bash ci/build.sh
 
+deploy:
+  - provider: script
+    script: PUBLISH=true bash ci/build.sh
```

Comment:
no need to specify `bash` here, just do:

```console
script: PUBLISH=true ci/build.sh
```

Reply:
Just copied it from `script` above. Should we remove it too?

Reply:
yep!

Reply:
Done.